doc/source/conf.py

```diff
@@ -56,7 +56,7 @@
 
 intersphinx_mapping = {
     'python': ('https://docs.python.org/3.6', None),
-    'numpy': ("https://numpy.org/doc/stable/", None),
+    'numpy': ("https://numpy.org/doc/1.19/", None),

```

Comment:
I would rather have seen us moving sphinx to a later python version but that is out fo scope of this PR

Reply:
That controls what version of the numpy documentation we point to in our types.